diff --git a/lib/esm/index.js b/lib/esm/index.js
index 14152f13d1fcc29be23c04a7f3ddbc3228251908..6a70ac41648e4430b415a4d2d88a06e655db61d1 100644
--- a/lib/esm/index.js
+++ b/lib/esm/index.js
@@ -380,7 +380,7 @@ const dleq = {
     proove(keyScalar, nonce, pubPoint, t, input, output) {
         t.protoName('DLEQProof');
         t.commitPoint('vrf:h', input);
-        const r = t.witnessScalar(`proving${'\0'}0`, [nonce]);
+        const r = t.witnessScalar('proving' + '\0' + '0', [nonce]);
         const R = RistrettoPoint.BASE.multiply(r);
         t.commitPoint('vrf:R=g^r', R);
         const Hr = input.multiply(r);
diff --git a/lib/index.js b/lib/index.js
index dd03166a9c35afc4f0a1754203ffcf1084f137f0..c2f24d933dd928c7a8076bbad46f3cc4f15695ad 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -389,7 +389,7 @@ const dleq = {
     proove(keyScalar, nonce, pubPoint, t, input, output) {
         t.protoName('DLEQProof');
         t.commitPoint('vrf:h', input);
-        const r = t.witnessScalar(`proving${'\0'}0`, [nonce]);
+        const r = t.witnessScalar('proving' + '\0' + '0', [nonce]);
         const R = ed25519_js_1.RistrettoPoint.BASE.multiply(r);
         t.commitPoint('vrf:R=g^r', R);
         const Hr = input.multiply(r);
